pages/linux/localectl.md

```diff
@@ -0,0 +1,20 @@
+# localectl
+
+> Control the system locale and keyboard layout settings.
+> More information: <https://www.freedesktop.org/software/systemd/man/localectl.html>.
+
+- Show current settings of the system locale and keyboard mapping:
+
+`localectl`
+
+- List available locales:
+
+`localectl list-locales`
+
+- Set a system locale variable:
+
+`localectl set-locale {{LANG}}={{en_US.UTF-8}}`
```

Comment:
```suggestion
`sudo localectl set-locale {{LANG}}={{en_US.UTF-8}}`
```

Reply:
Is the sudo necessary since localectl will prompt you for the password anyways?

Reply:
In may case it opens the GUI dialog, which is a bit annoying when you're working in the terminal, but that's probably just my setup. Generally, I think we should still include it, just to be safe.

Reply:
Timedatectl exhibits the same behavior and it doesn't have sudo listed if we want to go by precedent. 
I personally would like to put as little as possible in the example command as long as it works. The simpler the better.

Reply:
Sure, then let's leave it out.